db/seeds.rb

```diff
@@ -25,14 +25,73 @@
 puts "created #{User.count} new users"
 
 # Added 10 artworks
-10.times do
-  category = %w[painting drawing sculpture graphic_design seramic photography]
-  file = URI.open('https://source.unsplash.com/random')
-  artwork = Artwork.new(name: Faker::Name.name, category: category.sample, rate: rand(20..500))
-  artwork.user = User.all.sample
-  artwork.photos.attach(io: file , filename: 'random.png', content_type: 'image.png')
-  artwork.save!
-end
+# 10.times do
+#   category = %w[painting drawing sculpture graphic_design seramic photography]
+#   file = URI.open('https://source.unsplash.com/random')
+#   artwork = Artwork.new(name: Faker::Name.name, category: category.sample, rate: rand(20..500))
+#   artwork.user = User.all.sample
+#   artwork.photos.attach(io: file , filename: 'random.png', content_type: 'image.png')
+#   artwork.save!
+# end
+first_artwork = Artwork.new(name: "Mona Lisa", description: "A half-length portrait painting by Italian artist Leonardo da Vinci. 
+Considered an archetypal masterpiece of the Italian Renaissance, it has been described as 'the best known, the most visited, 
+the most written about, the most sung about, the most parodied work of art in the world'. 
+The painting's novel qualities include the subject's enigmatic expression, the monumentality of the composition, 
+the subtle modelling of forms, and the atmospheric illusionism.", rate: 11235, category: "painting")
+first_artwork.user = User.all.sample
+
```

Comment:
For the image upload, you need to paste in the following code for every artwork, for every image
`Cloudinary::Uploader.upload("image_filepath")`
For this to work however, you need to paste in the following  into an .env file (make new one if you don't have one already): CLOUDINARY_URL=cloudinary://255862718666814:p-hWaoynymeFG6-6JrekYOOkZ0I@henverscloud
Also, don't forget to attach the image to the artwork instance before saving it!
As so: `artwork.photos.attach(io: file , filename: 'random.png', content_type: 'image.png')`